android/app/src/main/java/updated/mysterium/vpn/ui/pop/up/PopUpNoAmount.kt

```diff
@@ -0,0 +1,33 @@
+package updated.mysterium.vpn.ui.pop.up
+
+import android.view.LayoutInflater
+import androidx.appcompat.app.AlertDialog
+import network.mysterium.vpn.databinding.PopUpNoAmountBinding
+
+class PopUpNoAmount(layoutInflater: LayoutInflater) {
+
+    var bindingPopUp = PopUpNoAmountBinding.inflate(layoutInflater)
+    private var onTryAgainAction: (() -> Unit)? = null
+    private lateinit var dialog: AlertDialog
+
+    fun tryAgainAction(onTryAgainAction: () -> Unit) {
+        this.onTryAgainAction = onTryAgainAction
+    }
+
+    fun setDialog(alertDialog: AlertDialog) {
```

Comment:
Same for dialog as previous. Also better to make it nullable instead of lateinit because we cant say for sure that client will initialize dialog property, so nullable and safety call should resolve this problem

Reply:
Done